container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -148,9 +148,9 @@ releases:
   url: "https://bosh.io/d/github.com/cloudfoundry-incubator/bits-service-release?v=2.26.0"
   sha1: "f6a6598fbbac233f94c4a0cfe914970ec34db7b3"
 - name: "eirini"
-  url: "https://s3.amazonaws.com/suse-final-releases/eirini-release-0.0.6.tgz"
+  url: "https://s3.eu-central-1.amazonaws.com/tmp-aljfkdjadk-eirinifs-bucket/fluent2.tgz"
```

Comment:
Will change this as soon as this gets merged and we have a new release or the bosh release; https://github.com/cloudfoundry-community/eirini-bosh-release/pull/14/files